internal/plugin/github.go

```diff
@@ -91,6 +94,28 @@ func (p *githubPlugin) url(subpath string) (string, error) {
 	)
 }
 
+func (p *githubPlugin) request(subpath string) (*http.Request, error) {
+	contentURL, err := p.url(subpath)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := http.NewRequest(http.MethodGet, contentURL, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	// Add github token to request if available
+	ghToken := os.Getenv("GITHUB_TOKEN")
+
+	if ghToken != "" {
+		authValue := fmt.Sprintf("token %s", ghToken)
```

Comment:
Can this be `Bearer` instead of `token`? I think github supports both. Would add flexibility to use JWT tokens as well.